egs/swbd/s5c/local/chain/tuning/run_tdnn_blstm_1b.sh

```diff
@@ -0,0 +1,254 @@
+#!/bin/bash
+
+# tdnn_blstm_1b is same as tdnn_blstm_1b, but with the more tdnn layers
```

Comment:
The AMI recipe change is helpful as it uses new type of lstm layers. I will separate those scripts into a new PR.